docs/plugins/repository-s3.asciidoc

```diff
@@ -190,8 +190,12 @@ The following settings are supported:
 
 `storage_class`::
 
-    Sets the S3 storage class type for the backup files. Values may be
+    Sets the S3 storage class for snapshot repository objects. Values may be
     `standard`, `reduced_redundancy`, `standard_ia`. Defaults to `standard`.
+    All new objects created will use the set storage class.
+    Changing this setting on an existing repository only affects the storage class for newly created objects,
+    resulting in a mixed usage of storage classes. Alternatively, S3 Lifecycle Policies can be used 
+    to manage storage class.
```

Comment:
I think it'd be better to say "the storage class of existing objects" instead of just "storage class" here.